pandas/tests/reshape/concat/test_concatenate.py

```diff
@@ -0,0 +1,1519 @@
+from collections import abc, deque
+from datetime import datetime
+from decimal import Decimal
+from io import StringIO
+
+import dateutil
+import numpy as np
+from numpy.random import randn
+import pytest
+
+from pandas.core.dtypes.dtypes import CategoricalDtype
+
+import pandas as pd
+from pandas import (
+    Categorical,
+    DataFrame,
+    DatetimeIndex,
+    Index,
+    MultiIndex,
+    Series,
+    Timestamp,
+    concat,
+    date_range,
+    read_csv,
+)
+import pandas._testing as tm
+from pandas.tests.extension.decimal import to_decimal
+
+
+@pytest.fixture(params=[True, False])
+def sort(request):
+    """Boolean sort keyword for concat and DataFrame.append."""
+    return request.param
+
+
+class TestConcatenate:
```

Comment:
i think this was a mistake originally, can you fold these into concat/test_concat.py